<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>847</width>
    <height>514</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Agregar Cuentas</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../tuvisa.png</normaloff>../tuvisa.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="Logo_2">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>181</width>
             <height>91</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">margin-right:8px</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>../tuvisa.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
           <property name="margin">
            <number>8</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_4">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLineEdit" name="AddEmail">
              <property name="styleSheet">
               <string notr="true">padding: 4px 4px 4px 16px;
color: rgb(0, 0, 127);
border-radius: 4px;
border: 1px solid rgb(0, 0, 127);</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
              </property>
              <property name="placeholderText">
               <string>Email</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="Consulado">
              <property name="styleSheet">
               <string notr="true">padding: 4px 4px 4px 16px;
color: rgb(0, 0, 127);</string>
              </property>
              <item>
               <property name="text">
                <string>Consulado 1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Mexico City</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Guadalajara</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Monterrey</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Ciudad Juarez</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Hermosillo</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Matamoros</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Merida</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Nogales</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Nuevo Laredo</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Tijuana</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_5">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QLineEdit" name="AddPassword">
              <property name="styleSheet">
               <string notr="true">padding: 4px 4px 4px 16px;
color: rgb(0, 0, 127);
border-radius: 4px;
border: 1px solid rgb(0, 0, 127);</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
              </property>
              <property name="placeholderText">
               <string>Password</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="Cas">
              <property name="styleSheet">
               <string notr="true">padding: 4px 4px 4px 16px;
color: rgb(0, 0, 127);</string>
              </property>
              <item>
               <property name="text">
                <string>Consulado 2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Mexico City</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Guadalajara</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Monterrey</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Ciudad Juarez</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Hermosillo</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Matamoros</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Merida</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Nogales</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Nuevo Laredo</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Tijuana</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_6">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QPushButton" name="BtnAgregarCuenta">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>28</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 127);
border-radius: 8px;</string>
              </property>
              <property name="text">
               <string>Guardar</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTableView" name="TablaCuentas">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="cursor" stdset="0">
            <cursorShape>UpArrowCursor</cursorShape>
           </property>
           <property name="tabletTracking">
            <bool>false</bool>
           </property>
           <property name="autoFillBackground">
            <bool>true</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);
</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QAbstractScrollArea::AdjustToContents</enum>
           </property>
           <property name="showGrid">
            <bool>true</bool>
           </property>
           <attribute name="horizontalHeaderCascadingSectionResizes">
            <bool>true</bool>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="ErrorMensaje">
     <property name="styleSheet">
      <string notr="true">color: rgb(170, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
